src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/ChatBoxPeripheral.java

```diff
@@ -349,11 +417,18 @@ public final MethodResult sendFormattedMessageToPlayer(@NotNull IArguments argum
     }
 
 
+    // 0 message, 1 title, 2 playerName, 3 prefix, 4 brackets, 5 bracket color, 6 range, 7 utf8compatible
```

Comment:
In 0.8 we must use a keyword table argument instead of the long long argument list :(